applications/embarrassingly-parallel.ipynb

```diff
@@ -384,7 +384,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We encourage you to watch the [dashboard's status page](../proxy/8787/status) to watch on going computation."
+    "We encourage you to watch the [dashboard's status page](http://127.0.0.1:8787) to watch on going computation."
```

Comment:
Does the previous link no longer work?

Reply:
I'm not sure, Sphinx warns that the file doesn't exist.